package.json

```diff
@@ -21,13 +21,12 @@
     "@shopify/tophat": "^0.0.4",
     "codecov": "^3.6.5",
     "eslint": "^6.8.0",
-    "eslint-plugin-prettier": "^3.1.2",
```

Comment:
Was this intentional?

Reply:
No! Good catch

Reply:
Ahhh yeah this is intentional, we use the shopify eslint plugin who contain this plugin.

Reply:
Nice deduping then 💯